src/script.js

```diff
@@ -5,32 +5,46 @@
 //HTML Elements
   const container1 = document.getElementById("result1Box");
   const container2 = document.getElementById("result2Box");
+  const form = document.querySelector("form");
 
 //Functions
   //Get User Input
 
+  form.addEventListener("submit", async (event) => {
+    event.preventDefault();
+    const formData = new FormData(form);
+     await pullFromAPI(shakespeare, "POST", "application/JSON", formData, container1);
+  })
+
   //Get Stuff from API
     async function pullFromAPI(api, method, contentType, requestBody, targetDiv) {
-      const url = api + requestBody;
-      console.log(url);
-      const translatedText = fetch((url), {
+      
+      // Turn form text input into string and create full URL
+      const formDataObject = Object.fromEntries(requestBody.entries());
+      const url = api;
+      const queryString = new URLSearchParams(formDataObject).toString();
+      const fullURL = url + "?" + queryString;
```

Comment:
Later on we can probs render this down to 

```js
let url = api;
url += ("?" + queryString);
```

or even just skip declaring url at all and going with

```js
api += ("?" + queryString);
```

This was definitely worth this depth of comment.